### PR TITLE
update macaw in klaviyo

### PR DESCRIPTION
**Pull request by lkostrowski**
## Scope of the PR

Zrzut ekranu 2023-06-21 o 17 27 26

## Checklist

- [ ] `.github/dependabot.yaml` is up-to date.
- [ ] I added changesets and [read good practices](/.changeset/README.md).

### CODE REVIEW

File: .changeset/proud-phones-change.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"saleor-app-klaviyo": minor
+---
+
+Rewritten app to use Macaw UI / next
```
Comment (peelar) [inline .changeset/proud-phones-change.md:5]: I think we can have a more user-centric changeset here. I suggest including information about how rewriting the app with the new Macaw makes it align visually with the rest of the dashboard.

File: apps/klaviyo/src/lib/ui/app-columns-layout.tsx [hunk 2]
```diff
@@ -1,21 +1,16 @@
...
-
-  return <div className={styles.root}>{children}</div>;
+export function AppColumnsLayout({ children }: PropsWithChildren<{}>) {
+  return (
+    <Box
+      display={"grid"}
+      __gridTemplateColumns={"280px auto 280px"}
+      gap={4}
+      __maxWidth={"1180px"}
+      __margin={"0 auto"}
```
Comment (peelar) [inline apps/klaviyo/src/lib/ui/app-columns-layout.tsx:11]:
```suggestion
      marginX={"auto"}
      marginY={0}
```

**Review by peelar — CHANGES_REQUESTED** [review]

File: .changeset/proud-phones-change.md (hunk 1 above)
Comment (lkostrowski) [reply .changeset/proud-phones-change.md]: Improved. But there are almost no changes so its hard describe anything interesting here

File: apps/klaviyo/src/lib/ui/app-columns-layout.tsx (hunk 2 above)
Comment (lkostrowski) [reply apps/klaviyo/src/lib/ui/app-columns-layout.tsx]: done

**Review by peelar — APPROVED** [review]

**Pull request merged**